recipes/pycalphad/meta.yaml

```diff
@@ -0,0 +1,80 @@
+# Note: there are many handy hints in commenets in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "0.4.1" %}
+{% set sha256 = "2603069927f4e6f69038f90a993f810c63585b6bf428473bcd5254d940a449f" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#   $ openssl sha256 <file>
+# On Windows, you'll need the openssl pacakge, available on conda-forge
+
+package:
+  name: pycalphad
+  version: {{ version }}
+
+source:
+  fn: pycalphad-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/pycalphad/pycalphad-{{ version }}.tar.gz
+  # md5 is OK but sha256 is desirable.
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    # When setuptools is available add the `--single-version-externally-managed --record record.txt` above.
+    - setuptools
+    - matplotlib
+    - pandas
+    - nose
+    - mock
+    - xarray !=0.8
+    - sympy
+    - pyparsing
+    - tinydb
+    - autograd
+    - scipy
+    - tqdm
+    - dask >=0.10
+    - distributed
+    - numpy >=1.9
+    - dill
+  run:
+    - python
+    - matplotlib
+    - pandas
+    - xarray !=0.8
+    - sympy
+    - pyparsing
+    - tinydb
+    - autograd
+    - scipy
+    - tqdm
+    - dask >=0.10
+    - distributed
+    - numpy >=1.9
+    - dill
+
+test:
+  # Python imports
+  imports:
+    - pycalphad
+    - pycalphad.core
+    - pycalphad.io
+    - pycalphad.plot
+
+about:
+  home: https://pycalphad.org/
+  license: MIT
+  summary: 'CALPHAD tools for designing thermodynamic models, calculating phase diagrams and investigating phase equilibria.'
+  doc_url: https://docs.pycalphad.org/
+  dev_url: https://github.com/pycalphad/pycalphad
+
+extra:
+  recipe-maintainers:
+    # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
+    - richardotis
+    - bocklund
```

Comment:
I am happy to be a maintainer for pycalphad.

Thank you,

Brandon Bocklund
brandonbocklund@gmail.com



> On Aug 8, 2016, at 8:44 PM, jakirkham <notifications@github.com> wrote:
> 
> In recipes/pycalphad/meta.yaml <https://github.com/conda-forge/staged-recipes/pull/1195#discussion_r73982122>:
> 
> > +    - pycalphad.core
> > +    - pycalphad.io
> > +    - pycalphad.plot
> > +
> > +about:
> > +  home: https://pycalphad.org/
> > +  license: MIT
> > +  summary: 'CALPHAD tools for designing thermodynamic models, calculating phase diagrams and investigating phase equilibria.'
> > +  doc_url: https://docs.pycalphad.org/
> > +  dev_url: https://github.com/pycalphad/pycalphad
> > +
> > +extra:
> > +  recipe-maintainers:
> > +    # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
> > +    - richardotis
> > +    - bocklund
> @bocklund <https://github.com/bocklund>, are you ok with being a maintainer on the pycalphad recipe in conda-forge <http://conda-forge.github.io/>?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/conda-forge/staged-recipes/pull/1195/files/98ecf6c84f20e33979c5d79c9a42608e591af785#r73982122>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AHU211VjHPfLh9jmH1LHfNEa3HRi-7ilks5qd81sgaJpZM4JfjnW>.
>